e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -222,6 +222,46 @@
               "createOptions": ""
             }
           },
+          "directMethodSender3": {
+            "version": "1.0",
+            "type": "docker",
+            "status": "running",
+            "restartPolicy": "always",
+            "env": {
+              "DirectMethodResultType": {
+                "value": "DirectMethodTestResult"
+              },
+              "DirectMethodName": {
+                "value": "ping"
+              },
+              "InvocationSource": {
+                "value": "Cloud"
+              },
+              "ServiceClientConnectionString": {
```

Comment:
Rename it to IOT_HUB_CONNECTION_STRING.

Reply:
The `deploymentTest1` is the only module that uses `IOT_HUB_CONNECTION_STRING` in Connectivity Test. Would you like me to change that one instead?